### PR TITLE
	Updates to Flit & Packet classes, adding get/set TrafficClass functions

### PR DESCRIPTION
**Pull request by ben-likes-unicorns**

### CODE REVIEW

**Comment by nicmcd** [comment]
@ben-likes-unicorns can you remove the setTrafficClass functions? I don't want people thinking that they can set the traffic class on a flit or packet.

File: src/types/Flit.cc [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 Flit::Flit(u32 _id, bool _isHead, bool _isTail, Packet* _packet)
     : id_(_id), head_(_isHead), tail_(_isTail), packet_(_packet),
-      sendTime_(U64_MAX), receiveTime_(U64_MAX) {}
+      vc_(U32_MAX), sendTime_(U64_MAX), receiveTime_(U64_MAX) {}
```
Comment (ben-likes-unicorns) [inline src/types/Flit.cc]: This is a change originated from Doug Gibson, but is being committed on my part.

**Comment by ben-likes-unicorns** [comment]
Sorry didn't get your comment about the setTrafficClass until I already merged on HPE GitHub.  I'll push these changes in another patch set there.

**Comment by nicmcd** [comment]
Changes have already been integrated

**Pull request closed without merging**